shaping/wrapping_test.go

```diff
@@ -2131,6 +2133,13 @@ func TestWrappingTruncationEdgeCases(t *testing.T) {
 			lastLine := lines[len(lines)-1]
 			lastRun := lastLine[len(lastLine)-1]
 			shouldTruncate := (tc.expectedTruncated > 0 || tc.forceTruncation)
+			// The VisualIndex and rune counts on the truncator will be populated during wrapping, and thus
```

Comment:
This seems confusing - we have tests documenting that data returned from the API is not to be used...
But nowhere in the API is this change explained.

Reply:
This comment is actually obsolete. I wrote it before realizing that we could still easily test these outputs for correctness. The two statements below modifying `trunc` populate its VisualIndex and Runes with correct expectations.

Reply:
I've pushed a change to drop this comment